scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -731,12 +731,30 @@ object FormatWriter {
     }
   }
 
-  private val trailingSpace = Pattern.compile("\\h+$", Pattern.MULTILINE)
+  /**
+    * No Unicode is necessary in the three (3) Pattern.compile() below.
+    * scalafmt parser will have already have objected.
+    * https://github.com/scala/scala/blob/2.11.x/spec/01-lexical-syntax.md
+    * gives the recognized scala whitespace characters.
+    * "Whitespace characters. \U0020 | \U0009 | \U000D | \U000A"
+    * (original text has lowercase u. U substituted here to allow scalafmt
```

Comment:
is this comment relevant to the code? :)

Reply:
I had thought so, but that section is superseded now.  After I commit, let me know if
I can improve the new text.   I try to get enough across so that I can remember six months
down the road and so that a colleague can read it without complaining about "War & Peace".